### PR TITLE
added animated drawer feature

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by l1llyp** [comment]
woooo!!!

File: boiler/src/client/components/App.jsx [hunk 1]
```diff
@@ -1,64 +1,384 @@
...
   );
+}
+
+
+
+
+
+
+
+// import React, { useState, useContext, useEffect } from 'react';
```
Comment (badams92) [inline boiler/src/client/components/App.jsx]: Do we really need all of this commented out code below?

**Comment by carlbenjaminlyon** [comment]
Looks good, just remove the comments and the app.bundle before the merge.

**Comment by badams92** [comment]
looks good

**Pull request merged**